### ISSUE TITLE
Buffer area around abandoned wells to sample the expected original land cover

### ISSUE THREAD

**Issue by ronnyhdez** [ronnyhdez/reclaimed_sites_ab#73]
According to ref #6

**Comment by ronnyhdez** [comment]
I have created first a buffer around the abandoned well polygons of 60m. Then I created a buffer around that first buffer of 200m.

Are those the buffer distances we want?

As a result, this is what I obtained:

![image](https://github.com/ronnyhdez/reclaimed_sites_ab/assets/24282285/e9b24c00-b024-4480-b8a6-11015b0f6c95)

The red squared polygon is the abandoned well polygon. The black section is the reference buffer (200m) and the space between the red abandoned well polygon and the reference polygon is the buffer area of 60m.
